lte/gateway/python/scripts/runtime_report.py

```diff
@@ -46,12 +46,15 @@ def merge_all_report(working_dir, list_xml_report_paths, output_path):
         xml_file_path = working_dir + "/" + xml_file_path
         test_result = ET.parse(xml_file_path)
         test_suites_data = test_result.getroot()
-
+        integration_tests = test_suites_data.attrib == {}
```

Comment:
I'm a bit confused by the these changes. So if we're running integration tests, there's an empty dict here and the real data is in the first element? Maybe it would be good to add a comment.

Reply:
When the xml is created with pytest then the structure is slightly different from what this script was intended to work with. If the attributes are missing, then we have to copy them from the child node.

Reply:
I see. :+1:

By the way, there's an unused variable in this function called `c` that you could remove if you like.